ssv/sync_committee_aggregator.go

```diff
@@ -95,11 +92,11 @@ func (r *SyncCommitteeAggregatorRunner) ProcessPreConsensus(signedMsg *SignedPar
 		anyIsAggregator = true
 
 		// fetch sync committee contribution
-		subnet, err := r.GetBeaconNode().SyncCommitteeSubnetID(indexes[i])
+		subnet, err := r.GetBeaconNode().SyncCommitteeSubnetID(r.GetState().StartingDuty.ValidatorSyncCommitteeIndices[i])
```

Comment:
not sure why putting it in the duty is better

Reply:
we getting this from the duty itself .
you used Prysm code but not from the api